eng/Versions.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
     
-    <VersionPrefix>2.9.12</VersionPrefix>
+    <VersionPrefix>2.9.13</VersionPrefix>
```

Comment:
I will create a separate PR to bump up the DogfoodAnalyzerVersion on this branch, seems there are quite a few RS diagnostics from 3.3.1 Roslyn.Diagnostics.Analyzers package.